plugins/node/opentelemetry-instrumentation-redis/src/types.ts

```diff
@@ -36,4 +37,43 @@ export interface RedisPluginClientTypes {
   address?: string;
 }
 
-export type RedisInstrumentationConfig = InstrumentationConfig;
+/**
+ * Function that can be used to serialize db.statement tag
+ * @param cmdName - The name of the command (eg. set, get, mset)
+ * @param cmdArgs - Array of arguments passed to the command
+ *
+ * @returns serialized string that will be used as the db.statement attribute.
+ */
+export type DbStatementSerializer = (
```

Comment:
Should we use `Redis` prefix here like for other types?

Reply:
I can definitely do that, but as mentioned below - my goal was to keep the same approach as in io-redis.